benefits/core/session.py

```diff
@@ -192,11 +201,24 @@ def uid(request):
     return u
 
 
-def update(request, agency=None, debug=None, eligibility_types=None, origin=None, token=None, token_exp=None, verifier=None):
+def update(
+    request,
+    agency=None,
+    auth=False,
```

Comment:
The default here should be `None`.

We may call `update()` after the user has successfully signed in (to update something else); if a value for `auth` wasn't given, the default of `False` would effectively sign the user out.

The logic correctly checks for a `bool` parameter, so `None` is the safer default here.